tests/console/yast2_i.pm

```diff
@@ -18,13 +18,34 @@ use testapi;
 use utils;
 use version_utils qw(is_tumbleweed is_jeos is_sle);
 
+sub set_action {
+    my $action = shift;
+    return unless defined($action);
```

Comment:
if call to this function does not make sense why we should "allow" it and produce dead code . let's die with error if someone will try to do so

Reply:
I am not sure what you mean with dead code. I did not want to put another if statement in that part of the code, therefore I chose to return when there is no argument passed to the function. 
I have seen this practice even in our code several times. :)

Reply:
the fact that you have seen something in our codebase is actually good reason to NOT to do it this way :) this codebase contains a lot of bad code ... 
when I was speaking about dead code I meant that with current implementation you can add `set_action();` anywhere in a code and it will remain there and "not bringing any harm" . But it is main thing about dead code from first view it is not bringing any harm but in a fact it does just not in direct way ...

Reply:
First of all, I am not talking only about our code base. And second of all, early return is just a matter a taste or coding style. :)